e2e-tests/functions

```diff
@@ -551,7 +551,7 @@ compare_kubectl() {
 			yq d -i ${new_result} 'metadata.generation'
 		fi
 	fi
-
+	cat ${new_result}
```

Comment:
Is this for permanent debugging purposes or temp left here?

Reply:
I deleted it) sorry my debug!